app/scripts/modules/titus/src/pipeline/stages/runJob/RunJobExecutionDetails.tsx

```diff
@@ -55,6 +55,17 @@ export class RunJobExecutionDetails extends React.Component<
     const jobId = cluster ? get(context['deploy.jobs'], cluster.region, [])[0] : null;
     const taskId = get(context, 'jobStatus.completionDetails.taskId');
 
+    const renderProperty = (entry: string) => {
+      const linkPattern = /^([a-zA-Z]{0,5})\:([^\s\n\t])*$/;
```

Comment:
```javascript
linkPattern.test(':foo') // true, probably shouldn't be?
linkPattern.test('a:b') // true, probably shouldn't be?
```
I feel like there are good example regexes for URLs that are pretty complex but also pretty thoroughly tested. It might be worth googling around a bit and seeing what's out there.

Reply:
The reason I wanted to keep it pretty open is to accommodate for our go links that comes in different shapes.

Reply:
I've spent days if not weeks of my life on linkification in the context of email clients 😆. In my experience trying to recognize any kind of URL that doesn't include a protocol is error prone and rarely works the way end users expect (much like Chris B's examples). Most clients (e.g. email or slack) that do this will exclusively look for protocols — our internal go links do not get linkified in most places without a protocol. There are some nicely tested regexes out there which will look for a whitelist of top level domains, but they're usually brittle and require updating whenever a new TLD is added.

If we wanted to special case go links without a protocol for our internal use, maybe we could consider a separate regex that matched on `go/` at the beginning of the string and put it in our internal build somehow?